client/jetpack-connect/authorize.js

```diff
@@ -1021,6 +1026,10 @@ export class JetpackAuthorize extends Component {
 				];
 			}
 
+			const companyName = this.isFromAutomatticForAgenciesClientPlugin()
+				? 'Automattic, Inc.'
+				: 'WordPress.com';
```

Comment:
Note: I haven't wrapped "WordPress.com" and "Automattic, Inc." here since they are company names being injected into an otherwise translatable string.